docs/book/src/tasks/certs/auto-rotate-certificates-in-kcp.md

```diff
@@ -0,0 +1,64 @@
+## Automatically rotating certificates using Kubeadm Control Plane provider
+
+When using Kubeadm Control Plane provider (KCP) it is possible to configure automatic certificate rotations. KCP does this by triggering a rollout when the certificates on the control plane machines are about to expire.
+
+If configured, rollout for certificate expiry is enabled for all new machines created using ClusterAPI v1.3 and above. To enable it for existing machines see [Enabling Certificate Expiry Rollout for Existing Machines](#enabling-certificate-expiry-rollout-for-existing-clusters)(#enabling-certificate-expiry-rollout-for-existing-clusters)
```

Comment:
```suggestion
If configured, rollout for certificate expiry is enabled for all new control plane machines created using ClusterAPI v1.3 and above. To enable it for existing control plane machines see [Enabling Certificate Expiry Rollout for Existing Machines](#enabling-certificate-expiry-rollout-for-existing-clusters)(#enabling-certificate-expiry-rollout-for-existing-clusters)
```
or KCP machines

Reply:
+ please update the link in this line as the headline was changed (machines instead of clusters)